Src/Web/Web.Shared.Net.Tests/RequestTrackingTelemetryModuleTest.cs

```diff
@@ -495,10 +521,10 @@ private string GetActivityRootId(string telemetryId)
             return telemetryId.Substring(1, telemetryId.IndexOf('.') - 1);
         }
 
-        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null)
+        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null, CorrelationIdLookupHelper correlationHelper = null)
```

Comment:
this will resolve the problem of initial requests. You can have an automation that will specify appID in configuration during the deployment